Extension/src/githubAPI.ts

```diff
@@ -218,10 +218,13 @@ export function getTargetBuild(builds: Build[], userVersion: PackageVersion, upd
     const targetVersion: PackageVersion = new PackageVersion(targetBuild.name);
     if (needsUpdate(userVersion, targetVersion)) {
         if (targetVersion.isEqual(userVersion)) {
```

Comment:
I don't think a check for `isEqual` should be needed here.  I believe `getTargetBuildInfo` will exit early if the versions are the same.

Reply:
I checked it, that's true!